packages/react-components/react-popover/src/popoverContext.ts

```diff
@@ -45,3 +45,43 @@ export type PopoverContextValue = Pick<
 
 export const usePopoverContext_unstable = <T>(selector: ContextSelector<PopoverContextValue, T>): T =>
   useContextSelector(PopoverContext, (ctx = popoverContextDefaultValue) => selector(ctx));
+
+export function usePopoverContextValues_unstable(state: PopoverState): PopoverContextValue {
```

Comment:
Let's move this change into a separate PR.
Also, I don't see this hook to be used anywhere in `Popover` code.